modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -16,36 +16,46 @@ Adjust RAM to match your latency requirements and the size and number of simulta
 
 == Minimum Software Requirements
 
-Mule requires minimum software resources:
+Mule requires the following minimum software resources:
 
-* Supported Java versions:
+* One of the following supported Java versions:
++
+** Oracle JDK 1.8.0 (JDK 1.8.0_161 recommended). 
++
+[NOTE]
+If you run a Standalone instance of Mule and Anypoint Studio freezes during start up, check to see if the JDK where Studio is installed is 1.8 Update 152 (1.8.0_152). If it is, downgrade to 1.8.0_151 to correct the problem.
```

Comment:
Now that we know that the recommended version for Mule standalone is JDK 1.8.0_172. The note is about a situation where the recommended *standalone* versions were _151 or _152.

Reply:
@dukesphere I don't understand what you're asking me to do in the last two notes.

Reply:
Two things:
1)  Need to know if the _152 issue  applies to the *GA* version of Studio. If issue does not exist in the GA version,  is there a need for the note? Asking because the note is in the EA release notes for Studio but not in subsequent RNs, so does it apply to GA versions? If people are using the EA (Early Access) version, shouldn't we be recommending that they use the GA version of Studio?
2) Is the advice to "downgrade" accurate anymore if the recommended JVM version for Mule EE standalone is 1.8.0_172, which is higher than _152? 

These questions need an answer to know what if any note is still needed on this topic, and the doc needs to be updated to reflect that 1.8.0_172 is the recommended version.